<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmGraphList</class>
 <widget class="QWidget" name="FrmGraphList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTableWidget" name="m_tblGraphs"/>
   </item>
   <item row="1" column="0">
    <spacer name="spacer3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>216</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnNewGraph">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>New Graph</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnDeleteGraph">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete Graph</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_tblGraphs</tabstop>
  <tabstop>btnNewGraph</tabstop>
  <tabstop>btnDeleteGraph</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
